gemsfx/src/main/java/com/dlsc/gemsfx/skins/PDFViewSkin.java

```diff
@@ -681,6 +681,17 @@ private void ensureVisible(Node node) {
             wrapper.setMaxWidth(Region.USE_PREF_SIZE);
             wrapper.setMaxHeight(Region.USE_PREF_SIZE);
             wrapper.rotateProperty().bind(pdfView.pageRotationProperty());
+            wrapper.addEventHandler(ScrollEvent.SCROLL, evt -> {
+                if (evt.isControlDown()) {
```

Comment:
Can you please replace this with eve.isShortcutDown()? On Mac CTRL + Scroll zooms the whole desktop (if enabled in system preferences).